src/libs/TransactionUtils.ts

```diff
@@ -0,0 +1,386 @@
+}
+
+/**
+ * Return the comment field (referred to as description in the App) from the transaction.
+ * The comment does not have its modifiedComment counterpart.
+ */
+function getDescription(transaction: Transaction): string {
+    return transaction?.comment?.comment ?? '';
+}
+
+/**
+ * Return the amount field from the transaction, return the modifiedAmount if present.
+ */
+function getAmount(transaction: Transaction, isFromExpenseReport: boolean): number {
+    // IOU requests cannot have negative values but they can be stored as negative values, let's return absolute value
+    if (!isFromExpenseReport) {
+        const amount = transaction?.modifiedAmount ?? 0;
+        if (amount) {
+            return Math.abs(amount);
+        }
+        return Math.abs(transaction?.amount ?? 0);
+    }
+
+    // Expense report case:
+    // The amounts are stored using an opposite sign and negative values can be set,
+    // we need to return an opposite sign than is saved in the transaction object
+    let amount = transaction?.modifiedAmount ?? 0;
+    if (amount) {
+        return -amount;
+    }
+
+    // To avoid -0 being shown, lets only change the sign if the value is other than 0.
+    amount = transaction?.amount ?? 0;
+    return amount ? -amount : 0;
+}
+
+/**
+ * Return the currency field from the transaction, return the modifiedCurrency if present.
+ */
+function getCurrency(transaction: Transaction): string {
+    const currency = transaction?.modifiedCurrency ?? '';
+    if (currency) {
+        return currency;
+    }
+    return transaction?.currency ?? CONST.CURRENCY.USD;
+}
+
+/**
+ * Return the merchant field from the transaction, return the modifiedMerchant if present.
+ */
+function getMerchant(transaction: Transaction): string {
+    return transaction?.modifiedMerchant ?? transaction?.merchant ?? '';
+}
+
+/**
+ * Return the waypoints field from the transaction, return the modifiedWaypoints if present.
+ */
+function getWaypoints(transaction: Transaction): WaypointCollection | undefined {
+    return transaction?.modifiedWaypoints ?? transaction?.comment?.waypoints;
+}
+
+/**
+ * Return the category from the transaction. This "category" field has no "modified" complement.
+ */
+function getCategory(transaction: Transaction): string {
+    return transaction?.category ?? '';
+}
+
+/**
+ * Return the billable field from the transaction. This "billable" field has no "modified" complement.
+ */
+function getBillable(transaction: Transaction): boolean {
+    return transaction?.billable ?? false;
+}
+
+/**
+ * Return the tag from the transaction. This "tag" field has no "modified" complement.
+ */
+function getTag(transaction: Transaction): string {
+    return transaction?.tag ?? '';
+}
+
+/**
+ * Return the created field from the transaction, return the modifiedCreated if present.
+ */
+function getCreated(transaction: Transaction): string {
+    const created = transaction?.modifiedCreated ?? transaction?.created ?? '';
```

Comment:
This caused a deploy blocker, the `??` operator only returns the right side of the operator if `transaction?. modifiedCreated ` is null but the `transaction?. modifiedCreated ` value in this case is `` an empty string so we return an empty string by mistake